templates/firewall.bash.j2

```diff
@@ -54,6 +54,14 @@ iptables -A INPUT -p tcp -m tcp --dport {{ port }} -j ACCEPT
 iptables -A INPUT -p tcp -m udp --dport {{ port }} -j ACCEPT
 {% endfor %}
 
+# Allowed hosts.
+{# Add a rule for each allowed host #}
+{% for host in firewall_allowed_hosts %}
+{% if 'ansible_eth0' in hostvars[host] %}
```

Comment:
I would rather just allow a list of IP addresses, rather than a list of potentially-complex dicts that requires such deep inspection. If someone just wanted to provide one or two IP addresses, the person would need to create a list like `hostvars[host]['ansible_eth0']['ipv4']['address'][] = '12.34.56.78'` just to get one host...